packages/@aws-cdk/aws-synthetics/test/test-class.test.ts

```diff
@@ -0,0 +1,26 @@
+import '@aws-cdk/assert/jest';
+import { App, Stack } from '@aws-cdk/core';
+import * as synthetics from '../lib';
+
+let stack: Stack;
+beforeEach(() => {
+  stack = new Stack(new App(), 'canaries');
+});
```

Comment:
If you do this, **always** add an `afterEach` that cleans up the mess (this'll save you from re-using the same `stack` if for some reason you fail to create a new one - e.g: if the instantiation failed due to lack of memory, or something).

It can save hours of troubleshooting weird bugs because of looking at the wrong place :)

Reply:
OK -- removing the `beforeEach()` to mitigate this issue. There isn't a clear cut pattern to clean up the stack it seems.